packages/usa-input-mask/src/test/input-mask-phone.template.html

```diff
@@ -0,0 +1,21 @@
+<div class="usa-input-masking">
+  <h3 class="site-preview-heading">US Telephone Number Input Mask</h3>
```

Comment:
You can prevent these `.site-preview-heading`'s from showing up in the code examples by adding them to the site's component include.

[Alert example with heading](https://github.com/uswds/uswds-site/blob/main/_includes/code/components/alert.html#L1)

How it currently looks
![image](https://user-images.githubusercontent.com/3385219/195121123-2edf2751-8d68-4013-9d96-6e6c2476244c.png)

Reply:
This has been reworked in uswds and uswds-site per the alert example you cited. In uswds, the component has examples that are now separated into multiple files, and in uswds-site we're using the .site-preview-heading to display the heading above each example.